src/med_bench/utils/classifiers.py

```diff
@@ -0,0 +1,423 @@
+from sklearn.linear_model import LogisticRegressionCV, RidgeCV, LassoCV
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.calibration import CalibratedClassifierCV
+import numpy as np
+from numpy.random import default_rng
+from scipy import stats
+import pandas as pd
+from pathlib import Path
+from scipy.stats import bernoulli
+from scipy.special import expit
+
+from itertools import combinations
+from sklearn.model_selection import KFold
+from sklearn.base import clone
+from .utils import get_interactions, _convert_array_to_R
+
```

Comment:
import can be better structures with isort

Reply:
This has been addressed in 1c272cd3d8fa44dddfc679ca99ba3221700fb6fa